src/oss/confluent.sh

```diff
@@ -686,6 +755,43 @@ config_service() {
     fi
 }
 
+enable_metrics_reporter() {
+    is_enterprise
+    status=$?
+    if [[ ${status} -ne 0 ]]; then
+        return 1
+    fi
+
+    local service="${1}"
+
+    local service_dir="${confluent_current}/${service}"
+    echo "metric.reporters=io.confluent.metrics.reporter.ConfluentMetricsReporter" \
+        >> "${service_dir}/${service}.properties"
+    echo "confluent.metrics.reporter.bootstrap.servers=localhost:${kafka_port}" \
+        >> "${service_dir}/${service}.properties"
+    echo "confluent.metrics.reporter.topic.replicas=1" >> "${service_dir}/${service}.properties"
+
+    return 0
+}
+
+enable_monitoring_interceptors() {
```

Comment:
Same as above, Will `enable_monitoring_interceptors` be run every single time confluent cli starts services? If yes, then would the echo statements below be redundant if it's run more than once? ie, the properties file will have each line written to it each time?

Reply:
Same as with metrics reporter.